### PR TITLE
https://issues.redhat.com/browse/ACM-14357 Update errata.adoc

### PR DESCRIPTION
**Pull request by swopebe**

### CODE REVIEW

File: release_notes/errata.adoc [hunk 1]
```diff
@@ -7,14 +7,18 @@ By default, Errata updates are automatically applied when released. The details
 
 See link:../install/upgrade_hub.adoc#upgrading-by-using-the-operator[Upgrading by using the operator] for more information about upgrades.
 
+== Errata 2.10.6
+
+* Fixes an issue with updates to the `ManagedCluster` object that prevented the hub template results from bring found with a search. (link:https://issues.redhat.com/browse/ACM-13370[ACM-13370])
```
Comment (dockerymick) [inline release_notes/errata.adoc]:
```suggestion
* Fixes an issue with updates to the `ManagedCluster` object that prevented the hub cluster template results from being found with a search. (link:https://issues.redhat.com/browse/ACM-13370[ACM-13370])
```

**Review by dockerymick — APPROVED** [review]
left a suggestion but approving. Thanks Brandi!

**Comment by swopebe** [comment]
Oh that is not a suggestion, that is a typo!! Thanks for catching that.

**Review by rhaignere — APPROVED** [review]
LGTM
approved, thank you both!

**Pull request merged**